code/_onclick/click.dm

```diff
@@ -22,13 +22,6 @@
 /atom/Click(location,control,params)
 	usr.ClickOn(src, params)
 
-/mob/living/Click()
-	if(isAI(usr))
-		var/mob/living/silicon/ai/A = usr
-		if(!A.aicamera.in_camera_mode) //Fix for taking photos of mobs
-			return
-	..()
-
```

Comment:
I tested the camera functionality and it worked fine after removing this, this code was added ~2016 because the AI camera wouldn't work when clicking mobs for some reason.